tests/System/client_multivo.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+testdir=`echo $PWD`
+
+JDLFILE="${testdir}/multiVO.jdl"
+cat > ${JDLFILE} <<EOF
+[
+Executable = "multiVOexe.sh";
+StdOutput = "job.log";
+StdError = "job.log";
+InputSandbox = "multiVOexe.sh";
+OutputSandbox = "job.log";
+Site = "LCG.UKI-LT2-IC-HEP.uk";
+JobName = "MultiVOTest";
+]
+EOF
+
+
+EXEFILE="${testdir}/multiVOexe.sh"
+cat > ${EXEFILE} <<EOF
+echo $DIRACSITE
+ghostname=`hostname --long 2>&1`
+gipname=`hostname --ip-address 2>&1`
+echo $ghostname "has address" $gipname
```

Comment:
Does this do what you intend? Or did you mean:
```suggestion
echo \$DIRACSITE
ghostname=\$(hostname --long 2>&1)
gipname=\$(hostname --ip-address 2>&1)
echo \$ghostname "has address" \$gipname
```